lib/common.sh

```diff
@@ -716,9 +716,6 @@ if command -v netstat &> /dev/null; then
     NETSTATCOMMAND="netstat"
 elif command -v ss &> /dev/null; then
     NETSTATCOMMAND="ss"
-else
-    eecho "[FATAL] Neither 'netstat' nor 'ss' is available!"
-    exit 1
 fi
```

Comment:
Since you have removed this from here, you should be checking in v4 specific place.
I don't see that, or did u already have it?

Reply:
I already had it in both watchdogv4 and nfsv4mountscript. Ex in nfsv4mountscript:

  if [ -z "$NETSTATCOMMAND" ]; then
        eecho "No socket statistics command (netstat or ss) found! Cannot proceed with TLS mount."
        exit 1
    fi